src/test/extension.test.ts

```diff
@@ -6,12 +6,16 @@ const sourcegraph = createMockSourcegraphAPI()
 // For modules importing Range/Location/URI/etc
 mock('sourcegraph', sourcegraph)
 
-import { decorateEditor, decorateLine, getDecorations } from '../extension'
+import { activate, decorateEditor, decorateLine, getDecorations } from '../extension'
 
 describe('extension', () => {
     it('works', () => void 0)
 })
 
+describe('check for extension activation', () => {
+    it('activate extension', () => expect(activate(sourcegraph.ExtensionContext)).toEqual(void 0))
```

Comment:
This works in that it checks that the extension's `activate` function does not crash when called. However it doesn't test much beyond that. I would add more comprehensive tests for the actual functionality, for instance:
- The extension reacts to windows/editor changes and decorates editors accordingly
- The extension reacts to configuration changes and decorates editors accordingly